src/assets/types/index.ts

```diff
@@ -19,11 +19,14 @@ export interface AssetInfoForChain extends AssetInfo {
 }
 
 export interface AssetConfig {
+  id: string;
   common_key: { [env: string]: string };
```

Comment:
since envs will be always the same can we use an object here?

```js
common_key: { 
  devnet: string,
  testnet: string,
  mainnet: string
};
```